config/mail.php

```diff
@@ -87,6 +87,8 @@
         'name' => env('MAIL_FROM_NAME', 'Example'),
     ],
 
+    'event_notification' => env('MAIL_EVENT_NOTIFICATION', 'webcontact@datamermaid.org'),
```

Comment:
So `webcontact@datamermaid.org` is just a default, if the env var doesn't exist? Would be nice if not anywhere in the code, but if just default that's ok.

Reply:
yep, it's a default value. Should be fine if we remove it but need to make sure the env var is there. I will update

Reply:
Sounds good. It should be sysadmin@datamermaid.org (I'll get that) -- and maybe an address at Catalyze as well?